components/automate-chef-io/Makefile

```diff
@@ -53,3 +52,14 @@ lint: themes/chef
 
 build: themes/chef generate_swagger
 	hugo
+
+# Consult scripts/spellcheck.sh in the root Automate directory for notes on using cspell.
+spellcheck:
```

Comment:
Glad to see you putting this in the makefile. It is long been on my "todo" list to eventually wire up spelling failures to fail the build, too.